.github/policies/resourceManagement.yml

```diff
@@ -192,5 +192,24 @@ configuration:
 
             You can refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md)
       description: Add breaking change instructions to PR.
+    - if:

```

Comment:
from https://eng.ms/docs/more/github-inside-microsoft/policies/index:

> Note: the file names are NOT significant. Name your policy files whatever you like. The location and content of the files and that they are YAML format is what matters.

So, we should not be stuffing every policy into a single, unmanageably large file. We should create a different file for each "thing", so they're easier to maintain.